pychunkedgraph/backend/chunkedgraph.py

```diff
@@ -1973,6 +1973,21 @@ def get_latest_roots(self, time_stamp: Optional[datetime.datetime] = get_max_tim
         return chunkedgraph_comp.get_latest_roots(self, time_stamp=time_stamp,
                                                   n_threads=n_threads)
 
+    def get_changed_roots(self,
```

Comment:
changed is a weird naming for this because ids never get _changed_
maybe get_root_id_changes()?

Reply:
changed to get_delta_roots